src/Page.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Input from './Input'
+import List from './List'
+
+
+function Page({input,list, onAdd, onComplete, onChange}) {
```

Comment:
```suggestion
function Page({
  input,list, onAdd, onComplete, onChange
}) {
```

린트를 돌리면 고쳐지겠지만 `props` 가 많은 경우 줄바꿈해주는 것도 코드 가독성에 좋아요!